### ISSUE TITLE
restrict user from creating organizations

### ISSUE THREAD

**Issue by arunabh123** [apiman/apiman#772]
is there a way to restrict users from creating new organizations in apimanui.

**Comment by volkflo** [comment]
Hi arunabh123,

We implemented this in our fork of apiman in the UI.

If I remember correctly we achieved this by hiding the Buttons if the user has not the role ‘apiadmin’.
The same applies for the methods to create a new organization.

I think we did not restricted this to the REST API of apiman.

Maybe, when I have time, I can check if I can contribute this functionality if the owners want me to contribute such a special functionality.
But I can not guarantee this contribution, because we modified the UI a lot.

Best regards,
Florian



Von: arunabh123 <notifications@github.com>
Gesendet: Tuesday, 5 November 2019 13:21
An: apiman/apiman <apiman@noreply.github.com>
Cc: Subscribed <subscribed@noreply.github.com>
Betreff: [apiman/apiman] restrict user from creating organizations (#772)


is there a way to restrict users from creating new organizations in apimanui.
